src/db/tables/onerep_scans.ts

```diff
@@ -40,21 +40,26 @@ async function getScanResults(
   return scanResults;
 }
 
-async function getLatestOnerepScanResults(
+async function getLatestOnerepScan(
   onerepProfileId: number | null,
-): Promise<LatestOnerepScanData> {
+): Promise<OnerepScanRow | null> {
   if (onerepProfileId === null) {
-    return {
-      scan: null,
-      results: [],
-    };
+    return null;
   }
 
   const scan = await knex("onerep_scans")
     .first()
     .where("onerep_profile_id", onerepProfileId)
     .orderBy("created_at", "desc");
 
+  return scan ?? null;
```

Comment:
Interesting! What does this do?
I thought the nullish coalescing would only return the right-hand side if the left-hand side was already null.

Reply:
Nullish coalescing returns the right-hand side if the left-hand side is `null` _or `undefined`_. So this ensures we return `null` if there was no result.